packages/components/psammead-navigation/src/Dropdown/index.test.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import {
+  CanonicalDropdown,
+  DropdownNavigationLi,
+  CanonicalHamburgerMenu,
+} from './index';
+import pidginNavData from '../../testHelpers/pidgin';
+
+describe('Navigation', () => {
+  shouldMatchSnapshot(
+    'dropdown should render correctly',
+    <CanonicalDropdown announcedText="Menu" onClose={() => {}}>
```

Comment:
Should probably have a similar test for `AmpDropdown`.

Reply:
The thing is that for now the only difference between `CanonicalDropdown` and an `AmpDropdown` is `on={onClose}` vs `onClick={onClose}` attribute. And this difference is not registered in a snapshot since it does not impact the appearance of the component. If I add a test for `AmpDropdown`, the snapshot will be exactly the same as the one I currently have for `CanonicalDropdown`. So it would not bring much value.

Reply:
Yes, good point!